### PR TITLE
Adds new wood and copper pikes with recipes.

### PR DESCRIPTION
**Pull request by Brian-Otten**
#### Summary
```SUMMARY: Content "Adds new wood and copper pikes with recipes."```

#### Purpose of change
the pike is a rather unique reach 3 medieval weapon, but it's quite hard for survivors to acquire this weapon because it requires smithing. This lets the survivor make the classic desperate variants of the pike, a wooden pike which only has a fire hardened point, and a pike with a crudely formed copper spearhead. These are inferior to the pike with steel spearhead but function similarly as weapons.

#### Describe the solution
Add wooden pike and copper pike and their recipes. Recipes are similar to the wooden spear and copper spear.

While i was in the files i also reduced W_block on all the pikes to 1, from 2. this makes it match all the other spears, because it was silly for the pike to be the best spear defensively when used up close.

PS: all the pikes have the looks_like pike flag to save tileset creators some headache.

### CODE REVIEW

File: data/json/items/melee/spears_and_polearms.json [hunk 1]
```diff
@@ -432,6 +432,46 @@
     "price_postapoc": 500,
     "qualities": [ [ "COOK", 1 ] ]
   },
+  {
+    "id": "pike_wood",
+    "type": "GENERIC",
+    "category": "weapons",
+    "name": { "ctxt": "weapon", "str": "wooden pike" },
+    "description": "This is a medieval weapon consisting of a wood shaft with a fire hardened point.",
+    "weight": "2300 g",
```
Comment (Night-Pryanik) [inline data/json/items/melee/spears_and_polearms.json:441]: This pike loses more than a kilogram after processing from a long stick (3.4 kg)?

**Comment by Night-Pryanik** [comment]
I'm not sure whether our `long stick` is long enough to provide a `REACH3` ability.

**Comment by Brian-Otten** [comment]
I made them the same weight as the normal pike, with the wooden one being slightly lighter due to not having a spearhead.
It's possible that all the pikes should weigh more, i'll run a quick audit.

I made them use the long stick for crafting because that's what the pike also uses, and because we don't have a "longer stick" item.

**Comment by Brian-Otten** [comment]
So historically pikes are between 2.5-6 kg in weight, but also 3-7.5 meters in length. Since the ingame pike is the shorter 3 meter variant it being on the low end of the weight scale seems okay.
https://en.wikipedia.org/wiki/Pike_(weapon)

**Comment by sayke** [comment]
Maybe long poles should drop from smashing saplings sometimes?

**Comment by Xenomorph-III** [comment]
Why is reach attack not available for the long stick itself? Or the quarterstaff? And if the long pole is used for pikes, shouldn't it have a reach of 3?

Just wondering.

**Pull request merged**